src/main/java/ch/njol/skript/expressions/ExprShootProperties.java

```diff
@@ -0,0 +1,166 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.util.slot.EquipmentSlot;
+import ch.njol.skript.util.slot.EquipmentSlot.EquipSlot;
+import ch.njol.skript.util.slot.Slot;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityShootBowEvent;
+import org.bukkit.inventory.ItemStack;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Shoot Properties")
+@Description("All the shooting properties of an <a href='events.html#entity_shoot_event'>entity shoot event</a>.")
+@Examples({
+	"on a player shooting a bow:",
+		"\tthe shot projectile was a tipped arrow of regeneration",
+		"\tthe name of the shooting bow is \"Example Bow\"",
+		"\tthe force of the arrow is 1.0 # Max",
+		"\tthe shooter was not {game::shooter}",
+		"\tcancel the event"
+})
+@Since("INSERT VERSION")
+public class ExprShootProperties extends SimpleExpression<Object> {
+
+	static {
+		Skript.registerExpression(ExprShootProperties.class, Object.class, ExpressionType.SIMPLE,
+				"[the] bow consumable",
+				"[the] shooting hand",
+				"should [the bow] consume item",
```

Comment:
I'm not a super big fan of this syntax. I might prefer an Effect+Condition combo.